google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditorTest.java

```diff
@@ -389,13 +389,12 @@ public void testDockerfileOverride() {
     assertEquals("an override", editor.getDockerfileTextField().getText());
   }
 
-  public void testDockerfileStartsDisabledAndWithModuleSetting() {
+  public void testDockerfileStartsDisabled() {
```

Comment:
Maybe the order counts.
I'm wondering if when the combo box model is changed, toggleDockerfileSection() is triggered by the item listener?